src/token/erc721/erc721.cairo

```diff
@@ -186,44 +206,95 @@ mod ERC721 {
+        }
+    }
+
+    /// Adds camelCase support for `IERC721`.
+    #[embeddable_as(ERC721CamelOnlyImpl)]
+    impl ERC721CamelOnly<
+        TContractState,
+        +HasComponent<TContractState>,
+        +SRC5::HasComponent<TContractState>,
+        +Drop<TContractState>
+    > of interface::IERC721CamelOnly<ComponentState<TContractState>> {
+        fn balanceOf(self: @ComponentState<TContractState>, account: ContractAddress) -> u256 {
+            self.balance_of(account)
+        }
+
+        fn ownerOf(self: @ComponentState<TContractState>, tokenId: u256) -> ContractAddress {
+            self.owner_of(tokenId)
+        }
+
+        fn getApproved(self: @ComponentState<TContractState>, tokenId: u256) -> ContractAddress {
+            self.get_approved(tokenId)
         }
 
         fn isApprovedForAll(
-            self: @ContractState, owner: ContractAddress, operator: ContractAddress
+            self: @ComponentState<TContractState>, owner: ContractAddress, operator: ContractAddress
         ) -> bool {
-            ERC721Impl::is_approved_for_all(self, owner, operator)
+            self.is_approved_for_all(owner, operator)
         }
 
-        fn setApprovalForAll(ref self: ContractState, operator: ContractAddress, approved: bool) {
-            ERC721Impl::set_approval_for_all(ref self, operator, approved)
+        fn setApprovalForAll(
+            ref self: ComponentState<TContractState>, operator: ContractAddress, approved: bool
+        ) {
+            self.set_approval_for_all(operator, approved)
         }
 
         fn transferFrom(
-            ref self: ContractState, from: ContractAddress, to: ContractAddress, tokenId: u256
+            ref self: ComponentState<TContractState>,
+            from: ContractAddress,
+            to: ContractAddress,
+            tokenId: u256
         ) {
-            ERC721Impl::transfer_from(ref self, from, to, tokenId)
+            self.transfer_from(from, to, tokenId)
         }
 
         fn safeTransferFrom(
-            ref self: ContractState,
+            ref self: ComponentState<TContractState>,
             from: ContractAddress,
             to: ContractAddress,
             tokenId: u256,
             data: Span<felt252>
         ) {
-            ERC721Impl::safe_transfer_from(ref self, from, to, tokenId, data)
+            self.safe_transfer_from(from, to, tokenId, data)
+        }
+    }
+
+    /// Adds camelCase support for `IERC721Metadata`.
+    #[embeddable_as(ERC721MetadataCamelOnlyImpl)]
+    impl ERC721MetadataCamelOnly<
+        TContractState,
+        +HasComponent<TContractState>,
+        +SRC5::HasComponent<TContractState>,
+        +Drop<TContractState>
+    > of interface::IERC721MetadataCamelOnly<ComponentState<TContractState>> {
+        fn tokenURI(self: @ComponentState<TContractState>, tokenId: u256) -> felt252 {
+            assert(self._exists(tokenId), Errors::INVALID_TOKEN_ID);
+            self.ERC721_token_uri.read(tokenId)
```

Comment:
Hi, I hope you don't mind me barging in here.

I've noticed that in other cases of a CamelOnly impls the code calls their non camel counterparts (e.g. `balanceOf` -> `balance_of`), but this is not the case here for `tokenURI`. I've spotted it by chance on `main` because I was adopting the implementation to my needs and saw that calling `token_uri` and `tokenURI` behave differently.

Wanted to open a PR, but then I saw this one. Do you think you can update it here for consistency's sake?

Reply:
Fixed! Thanks, @milancermak :)